clang.rst

```diff
@@ -221,7 +233,9 @@ Finally is ``make test`` (formatting added for clarity): ::
 
 If you are using the address sanitizer, its important to pipe the output through
 ``asan_symbolize.py`` to get a good trace. For example, from Issue 20953 during
-compile (formatting added for clarity): ::
+compile (formatting added for clarity):
+
+.. code-block:: none
```

Comment:
Why not console?

Reply:
Because the `console` lexer thinks the lines beginning `#` in the output are commands. (Traditionally, `#` is the prompt in a shell with root privilege.)

There's not much in it either way, but I went for less distracting.

Reply:
Ah!  Thanks, it is the best choice.